### PR TITLE
Fixed pydocs and other references to old tf.* classes from Graph Editor

### LINKED ISSUE #21: Remove references to `tf.Graph` from API docs.
There are still some references here and there in docstrings to `tf.Graph`, `tf.Tensor`, etc. from the days when this code base was `contrib.graph_editor` and directly modified TensorFlow's Python graph objects. Find the offending docstrings and fix them. Double-check that these references to TensorFlow internal classes are not attached to chunks of code that haven't been properly ported to operate over `GraphDef` protobufs instead of `tf.Graph` objects.

### PR DESCRIPTION
**Pull request by BryanCutler**
Fixes #21

### CODE REVIEW

File: graph_def_editor/reroute.py [hunk 1]
```diff
@@ -134,7 +134,7 @@ def _reroute_ts(ts0, ts1, mode, can_modify=None, cannot_modify=None):
   """Reroute the end of the tensors in each pair (t0,t1) in ts0 x ts1.
 
   This function is the back-bone of the Graph-Editor. It is essentially a thin
-  wrapper on top of the tf.Operation._update_input.
+  wrapper on top of the `gde.Node.replace_input`.
```
Comment (BryanCutler) [inline graph_def_editor/reroute.py]: I believe this is the equivalent, but can you verify @frreiss ?

Comment (frreiss) [reply graph_def_editor/reroute.py]: Yes, that's basically equivalent.
Nit: Remove "the" from this line or add the word "method" after `gde.Node.replace_input`.

**Comment by frreiss** [comment]
LGTM

File: graph_def_editor/reroute.py (hunk 1 above)
Comment (BryanCutler) [reply graph_def_editor/reroute.py]: sure, that would be correct

**Comment by BryanCutler** [comment]
Thanks @frreiss , I fixed the grammar mistake

**Comment by frreiss** [comment]
Merging.

**Pull request merged**